jekyll/_cci2/migrate-from-launch-agent-to-machine-runner-3.adoc

```diff
@@ -0,0 +1,88 @@
+---
+contentTags:
+  platform:
+  - Cloud
+---
+= Migrate from launch agent to machine runner 3.0 - Open preview
+:page-layout: classic-docs
+:page-liquid:
+:page-description: Steps to migrate from using launch agent to the machine runner 3.0 preview
+:icons: font
+:toc: macro
+:toc-title:
+:machine:
+:experimental:
+:linux:
+
+This page describes how to migrate an existing launch agent installation to machine runner 3.0. Machine runner 3.0 includes improved network resiliency, and a simpler installation management strategy, utilizing operating system packages.
+
+NOTE: Machine runner 3.0 is currently in **open preview**, and is currently available on **Linux** only
+
+Migrating from launch agent to machine runner 3.0 is a straightforward process. The prerequisites remain the same. First, uninstall and remove launch agent, then, install the machine runner 3.0 agent. The configuration file is 1:1 compatible between the agents so no modifications are needed during the migration
+
+[#uninstall-launch-agent]
+== 1. Uninstall launch agent
+
+The first step is to uninstall launch agent. The exact process depends on the platform, as described in the following sections.
+
+. If the agent has been installed as a `systemd` service, run the following commands to stop and uninstall the service unit, and delete the service file (the new agent will install a new service unit):
++
+```shell
+sudo systemctl stop circleci.service
+sudo systemctl disable circleci.service
+
+sudo rm /usr/lib/systemd/system/circleci.service
+```
+
+. The launch agent binary can now be removed from the machine. The default install location is `/opt/circleci/circleci-launch-agent`, but this may differ on your machine if the agent was installed somewhere else. The entire `/opt/circleci` subdirectory can be removed at this point.
+
+. Finally, move the agent configuration file. The default location for this is `/etc/opt/circleci/launch-agent-config.yaml`. Move the file to `/etc/circleci-runner/circleci-runner-config.yaml` using the following command:
++
+```shell
+mv /etc/opt/circleci/launch-agent-config.yaml /etc/circleci-runner/circleci-runner-config.yaml
+```
++
+Alternatively, you can let the machine agent install process create a config file template and copy the existing config into the newly created file (CROSSLINK)
+
+[#uninstalling-launch-agent-linux-se]
+=== RHEL/Rocky Linux
```

Comment:
Not clear on why this can't just be another step in this section rather than under a separate heading

Reply:
This step is specifically needed for RHEL/Rocky (formerly centOS but linux drama) distros. It won't break anything if someone tries to run it on a debian based system (ubuntu etc) but it will also give them an error that the program wasn't found on their system if they do try to run it there.